pkg/connector/ids/ids.go

```diff
@@ -119,6 +119,7 @@ const (
 	PeerTypeUser    PeerType = "user"
 	PeerTypeChat    PeerType = "chat"
 	PeerTypeChannel PeerType = "channel"
+	PeerTypeEmoji   PeerType = "emoji"
```

Comment:
Would be better to name the constant as FakePeerTypeEmoji or something to clarify it's fake